docs/pai-management/doc/how-to-add-new-conm.md

```diff
@@ -0,0 +1,200 @@
+
+
+#### Default Configuration <a name="Default"></a>
+
+###### Path
+
+Please create the default configuration in the following path. 
+
+```bash
+src/${service_name}/config/${service_name}.yaml
+```
+
+
+###### Format
+
+${service_name}.yaml
+```yaml
+# key : value
+service-a-key1: default—value1
+
+service-a-key2: default-value2
+
+
+service-a-key3:
+    service-a-key3-subkey1: default-value3-subvalue1
+    service-a-key3-subkey2: default-value3-subvalue2
+    service-a-key3-subkey3: default-value3-subvalue3
+    
+service-a-key4:
+    - default-value4-sub1
+    - default-value4-sub2
+
+```
+
+#### overwrite Configuration <a name="Overwrite"></a>
+
+- Reconfigure the default value
+- Some properties is mandatory, user will have to configure it. Please put those configuration into the overwrite configuraiton in service-configuration.yaml
+
+###### Update example/cluster-configuration
+
+You should add the overwrite section in [example/service-configuration.yaml](../../../examples/cluster-configuration/services-configuration.yaml)
+
+Add the overwrite section of ```${service_name}``` like the following. Note, you could omit the default value which exists in default configuraiton.
+
+```yaml
+${service_name}:
+  
+  service-mandatory-key1: example-value1
+
+  service-a-key1: default—value1
+
+  service-a-key2: default-value2
+
+  service-a-key3:
+      service-a-key3-subkey1: default-value3-subvalue1
+      service-a-key3-subkey2: default-value3-subvalue2
+      service-a-key3-subkey3: default-value3-subvalue3
+    
+  service-a-key4:
+      - default-value4-sub1
+      - default-value4-sub2
+  
+``` 
+
+###### Update deployment/quick-start/services-configuration.yaml.template
+
+You should add the overwrite section in [deployment/quick-start/services-configuration.yaml.template](../../../deployment/quick-start/services-configuration.yaml.template)
+
+Add the overwrite section of ```${service_name}``` like the following. Please omit the default value, and only left the mandatory value.
+
+```yaml
+${service_name}:
+    service-mandatory-key1: usable-value1
+    
+```
+
+#### Service Parser <a name="Service"></a>
+
+###### path
+
+Please create the service.parser in the following path. 
+
+```bash
+src/${service_name}/config/${service_name}.py
+```
+###### Specification
+
+You should implement the parse class like following.
+
+```python
+class cluster:
+
+    def __init__(self, cluster_configuration, service_configuration, default_service_configuraiton):
+        None
+    
+    #### Fist check, ensure all the configured data in cluster_configuration, service_configuration, default_service_configuration is right. And nothing is miss.
+    def validation_pre(self):
+        #   if error: 
+        #      return False, "message"
+
+        return True, None
+    
+    #### Generate the final service object model
+    def run(self):
+        # parse your service object model here, and return a generated dictionary
+        return cluster_com
```

Comment:
A question, origin ObjectModel contains conversion from `vc` to `hadoopQueue`, now I need to move this logic to the parser?